### PR TITLE
Fix sorting of percentage values

### PR DESCRIPTION
**Pull request by CrypticWizard**
This PR is in response to issue #1013 

The percentage value was being calculated in the firstUpdated() method, hence didn't update after the first insert. **I wasn't able to test this code though because properties data is not available locally**.

### CODE REVIEW

File: data/devrel_team.yaml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+name: Adam Argyle
```
Comment (jrobbins) [inline data/devrel_team.yaml:1]: This doesn't belong in this PR.

File: static/elements/chromedash-x-meter.js [hunk 2]
```diff
@@ -42,19 +42,15 @@ class ChromedashXMeter extends LitElement {
...
   showTimeline() {
     window.location.href = this.href;
   }
 
   render() {
     return html`  
-      <div style="width: ${(this.value / this.max * 100)}%">  
-        <span>${this.valueFormatted}</span> 
+      <div @click="${this.showTimeline}" style="width: ${(this.value / this.max * 100)}%">  
+        <span>${this.value <= 0.000001 ? '<=0.000001%' : this.value + '%'}
```
Comment (jrobbins) [inline static/elements/chromedash-x-meter.js:52]: Since this code no longer uses this.valueFormatted, you should delete line 9.

**Review by jrobbins — CHANGES_REQUESTED** [review]

File: data/devrel_team.yaml (hunk 1 above)
Comment (CrypticWizard) [reply data/devrel_team.yaml]: Done!

File: static/elements/chromedash-x-meter.js (hunk 2 above)
Comment (CrypticWizard) [reply static/elements/chromedash-x-meter.js:51]: Done!

**Review by jrobbins — APPROVED** [review]
